src/main/java/org/jabref/gui/util/ValueTableCellFactory.java

```diff
@@ -100,7 +101,13 @@ protected void updateItem(T item, boolean empty) {
                     if (toTooltip != null) {
                         String tooltipText = toTooltip.apply(rowItem, item);
                         if (StringUtil.isNotBlank(tooltipText)) {
-                            setTooltip(new Tooltip(tooltipText));
+                            Screen currentScreen = Screen.getPrimary();
+                            double maxWidth = currentScreen.getBounds().getWidth();
+                            Tooltip tooltip = new Tooltip(tooltipText);
+                            tooltip.setMaxWidth(maxWidth * 2 / 3);
+                            // tooltip.setTextOverrun(OverrunStyle.CENTER_ELLIPSIS); // info: can be enabled instead of wrapping the text
```

Comment:
This comment can be removed, right?

Reply:
Indeed, I prefer wrapping anyways. Removed.